app-crypt/paperkey/paperkey-1.4.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Tool to extract secret bits from a secret key to be printed on paper"
```

Comment:
Sorry, I just noticed that this line is > 70 chars, so it breaks the [recommendations for SUMMARY](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#summary).
Maybe we could use this:
```
SUMMARY="A tool to backup/restore OpenPGP keys to/from printed paper"
```
If you have something better or with "secret bits", please feel free to suggest ;-)